presto-test-jdbc-compatibility/src/test/java/io/prestosql/TestingServerUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql;
+
+import io.prestosql.server.testing.TestingPrestoServer;
+
+import java.lang.reflect.Field;
+
+public class TestingServerUtils
+{
+    private TestingServerUtils() {}
+
+    public static void setTestingServer(Object target, TestingPrestoServer server)
+    {
+        try {
+            Field serverField = target.getClass().getSuperclass().getDeclaredField("server");
```

Comment:
We should refactor TestJdbcResultSet into
base class and actual class and avoid reflection.

Reply:
Yeah, but I cannot do this for the old TestJdbcResultSet tests (from previous releases)

Reply:
Add a short comment explaining why